lib/jekyll/filters.rb

```diff
@@ -373,8 +373,8 @@ def time(input)
       when Numeric
         Time.at(input)
       else
-        Jekyll.logger.error "Invalid Date:", "'#{input}' is not a valid datetime."
-        exit(1)
+        raise Errors::InvalidDateError,
+          "Invalid Date: '#{input}' is not a valid datetime."
```

Comment:
Adding the `.inspect` call means instead of `Invalid Date: "" is not a valid datetime.`, you'd see `Invalid Date: "nil" is not a valid datetime.`